packages/webapi/src/lib/fetch.ts

```diff
@@ -1,102 +0,0 @@
-import { bootstrap as bootstrapGlobalAgent } from 'global-agent'
-import type { RequestInit } from 'node-fetch'
-import { default as nodeFetch, Headers, Request, Response } from 'node-fetch'
-import Stream from 'node:stream'
-import * as _ from './utils'
-
-bootstrapGlobalAgent({
-	environmentVariableNamespace: '',
-})
-
-export { Headers, Request, Response }
-
-export const fetch = {
-	fetch(
-		resource: string | Request,
-		init?: Partial<FetchInit>
-	): Promise<Response> {
-		const resourceURL = new URL(
-			_.__object_isPrototypeOf(Request.prototype, resource)
-				? (resource as Request).url
-				: _.pathToPosix(resource),
-			typeof Object(globalThis.process).cwd === 'function'
-				? 'file:' + _.pathToPosix(process.cwd()) + '/'
-				: 'file:'
-		)
-
-		if (resourceURL.protocol.toLowerCase() === 'file:') {
```

Comment:
I agree with removing this custom `file:` behavior, but let's make sure we note this removal.

Reply:
I think this would be a `major` changeset for `@astrojs/webapi`